### PR TITLE
elcontracts writer tests

### PR DESCRIPTION
**Pull request by pablodeymo**

### CODE REVIEW

File: crates/chainio/clients/elcontracts/src/writer.rs [hunk 1]
```diff
@@ -256,4 +269,57 @@ mod tests {
...
+        let receipt = ANVIL_RPC_URL
+            .get_transaction_receipt(tx_hash)
+            .await
+            .unwrap();
+        assert!(receipt.unwrap().status());
+
+        // FIX this:
+        //     Transaction: 0xb497b163bef1f1f23dcc6bdc1cbaae653cb438dd3244940016c07a62642a8591
+        //     Gas used: 32928
+        //     Error: reverted with: revert: DelegationManager.registerAsOperator: caller is already actively delegated
```
Comment (supernovahs) [inline crates/chainio/clients/elcontracts/src/writer.rs:319]: @pablodeymo 
this is due to this
https://github.com/Layr-Labs/eigensdk-rs/blob/749f634d192cbeea014b29d7a1040b7abbd3f660/crates/contracts/script/RegisterOperatorsWithEigenlayer.s.sol#L63

We register some anvil keys in scripts here already on eigen layer.

**Comment by supernovahs** [comment]
https://github.com/Layr-Labs/eigensdk-rs/pull/80  
The go sdk had some additional functions, which need updated bindings and submodule. Did some of that above for your ref
@pablodeymo

File: crates/chainio/clients/elcontracts/src/writer.rs (hunk 1 above)
Comment (pablodeymo) [reply crates/chainio/clients/elcontracts/src/writer.rs]: this was already fixed, thanks 👍

**Comment by pablodeymo** [comment]
> #80 The go sdk had some additional functions, which need updated bindings and submodule. Did some of that above for your ref @pablodeymo

I think we can finish this PR as is, and the we can open a new PR with the additional functions you put in #80:
- `set_claimed_for`
- `process_claim`
- `force_deregister_from_operator_set`
- `set_operator_commission_bips`

**Review by supernovahs — APPROVED** [review]

**Pull request merged**